### PR TITLE
QA: Add padding to paragraphs in Overlay

### PR DESCRIPTION
**Pull request by chrismekim**
@hhff There should be some padding between the paragraphs, right?

Before:
Screen Shot 2020-04-07 at 7 12 33 PM

After: 
Screen Shot 2020-04-07 at 7 13 42 PM

Preview: https://share.getcloudapp.com/qGuo9BbD#

### CODE REVIEW

File: src/styles/components/Overlay.scss [hunk 1]
```diff
@@ -34,6 +34,11 @@
     letter-spacing: -5;
 
     @include media("md-up") {
+      font-size: 2.5rem;
```
Comment (chrismekim) [inline src/styles/components/Overlay.scss:37]: Added because the font is huge around `md` screens and gets cut off.

**Comment by hhff** [comment]
Looks good!

@chrismekim can you add a `HackerConsole.enableNightmode` method so I can preview in staging before 7pm?

**Comment by chrismekim** [comment]
@hhff Added enableNightmode!

**Comment by hhff** [comment]
thank you! pls deploy to staging :)

**Pull request merged**